pricing-page-exercise/salih/styles.css

```diff
@@ -0,0 +1,195 @@
+/* Global */
+
+*, ::after, ::before {
+    box-sizing: border-box;
+    font-family:sans-serif;
+}
+ul{
+    list-style-type: none;
+    padding: 0;
+    text-align: center;
+}
+.main li{
+    padding: 0.2rem 0;
+}
+h1{
+    margin: 0;
+    text-align: center;
+    margin-top: 5px;
+}
+button{
+    margin: 15px 10px;
+    min-width: 60%;
+    padding: 12px 0;
+}
+small{
+    font-weight: 100;
+    font-size: 0.5em;
+}
+.blue-button{
+    background-color: #007bff;
+    color: white;
+    border: none;
+    border-radius: 5px;
+    font-size: 1em;
+}
+
+.white-button{
+    color: #007bff;
+    background-color: white;
+    border: 1px solid #007bff;
+    border-radius: 5px;
+    font-size: 1em;
+}
+
+
+/*_____________________________________*/
+/*Navigation Bar*/
+
+.nav-container{
+    width: 100%;
+    min-height: 50px;
+    background-color: white;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    border-bottom: 1px solid #dee2e6;
+    box-shadow: 0 0.25rem 0.75rem rgba(0, 0, 0, .05);
+}
+.menu-container{
+    display: flex;
+    align-items: center;
+    min-width: 35%;
+    justify-content: center;
+    /* border: 1px solid red; */
```

Comment:
It's better to always remove commented out code from your code however for the learning purposes it's ok to leave it in your personal projects.

Reply:
I have found a few more and deleted them.